qiime2/sdk/plugin_manager.py

```diff
@@ -47,11 +46,17 @@ def __new__(cls):
 
     def _init(self):
         self.plugins = {}
-        self.semantic_types = {}
+        self.type_fragments = {}
         self.transformers = collections.defaultdict(dict)
+        self._reverse_transformers = collections.defaultdict(dict)
         self.formats = {}
         self.views = {}
         self.type_formats = []
+        self._ff_to_sfdf = {}
+        self._importable = set()
+        self._exportable = set()
+        self._canonical_formats = set()
+        self.types = {}
```

Comment:
Consider the order of these instance attributes, there's so many some comments would be nice. For example, public attributes could be at the top, with the private ones after a line break.